gibberish-generation/README.md

```diff
@@ -0,0 +1,47 @@
+# G2P (Grapheme-to-Phoneme) Model
+
+This repository provides a neural network-based grapheme-to-phoneme (G2P) conversion model using PyTorch and spaCy, along with additional utilities for training and inference.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Model Usage](#setup)
+- [Model Training](#checking-pytorch-with-cuda)
+
+## Installation
+
+To run this project, you need to install the required Python packages. Follow the steps below to set up the environment:
+
+### Prerequisites
+
+- Python 3.8+
+- NVIDIA GPU with CUDA and Tensor Cores (for training)
+  - If your GPU does not have CUDA Cores, PyTorch will automatically perform model training on your CPU.
+  - If your GPU does not have Tensor Cores, you cannot use [Mixed Precision Training](https://docs.nvidia.com/deeplearning/performance/mixed-precision-training/index.html) and references to `autocast` and `GradScalar` will need to be removed.
+
+### Dependencies
+
+```
+pip install torch
```

Comment:
We have a requirements.txt, activate the venv using the readme in root of the project, install fhe new dependencies then pip freeze the requirements.txt file

Also, make sure to follow the commit conventions for next time. 

git commit -m "#tktnum: message"

Since once we delete the branch we will have no connection back to the ticket. Having it in comments will auto link